web/packages/teleport/src/TopBar/TopBar.tsx

```diff
@@ -165,27 +155,23 @@ export function TopBar({ CustomLogo }: TopBarProps) {
         </ButtonIconContainer>
       )}
       <Flex height="100%" alignItems="center">
-        {!hasDockedElement && assistEnabled && (
-          <ButtonIconContainer onClick={() => setShowAssist(true)}>
-            <BrainIcon />
+        {!hasDockedElement && assistProps?.assistEnabled && (
+          <ButtonIconContainer onClick={() => assistProps?.setShowAssist(true)}>
+            <BrainIcon
+              size={currentWidth >= theme.breakpoints.medium ? 24 : 20}
+            />
```

Comment:
missing `color={selected ? 'text.main' : 'text.muted'}` property as other top nav buttons?

Reply:
also is this how it worked before where the brain icon disappears after cliking on it? (it just feels a little strange in the demo that the icon disappears since the assist doesn't cover the topbar anymore)

Reply:
Yeah no logic has changed with the display of the brain icon, it hides when showAssist is true. 

Also, for the colors, Yassine is adding in a new icon for this specifically and updating the colors there (separate PR)